shared/bus.go

```diff
@@ -143,3 +155,11 @@ func (m *bus) GetConsensusModule() modules.ConsensusModule {
 func (m *bus) GetTelemetryModule() modules.TelemetryModule {
 	return m.telemetry
 }
+
+func (m *bus) GetConfig() map[string]json.RawMessage {
+	return m.config
+}
+
+func (m *bus) GetGenesis() map[string]json.RawMessage {
+	return m.genesis
```

Comment:
Same comment as above, i.e. this "leaks" the genesis.
Also need to use a non-pointer receiver for the method for a Get method.